sbom/examples/container_image/release/from_catalog.py

```diff
@@ -181,6 +189,43 @@ def generate_sboms_for_image(image_nvr):
             image_pkg["externalRefs"].append(ref)
         per_arch_images.append(image_pkg)
 
+        # Add in parent images
+        parent_images = koji_session.getBuild(image_nvr)
+        for key in ("extra", "typeinfo", "image", "parent_images"):
+            parent_images = parent_images.get(key, {})
+
+        parent_images = [img.rsplit("/")[-1] for img in parent_images if img != "scratch"]
+        direct_parent_index = len(parent_images) - 1
+        for index, parent_image in enumerate(parent_images):
+            parent_spdx_id = f"SPDXRef-parent-image-{index}-{image['architecture']}"
+            parent_pkg = {
+                "SPDXID": parent_spdx_id,
+                "name": f"{parent_image}_{image['architecture']}",
+                "versionInfo": "NOASSERTION",
+                "supplier": "Organization: Red Hat",
+                "downloadLocation": "NOASSERTION",
+                "licenseDeclared": "NOASSERTION",
+                "externalRefs": [],
```

Comment:
Are we able to create purls here for the parent images?

Reply:
Added